lib/rake/ci.rake

```diff
@@ -77,3 +85,18 @@ desc 'Update the server as part of continuous integration.'
 task :ci do
   HipChat.wrap('CI build') { Rake::Task[rack_env?(:test) ? 'ci:test' : 'ci:all'].invoke }
 end
+
+def upgrade_frontend(name, host)
```

Comment:
Please document the method parameters (it isn't obvious from the signature that `host` is a string) and that it returns a boolean indicating whether the upgrade was successful.